src/mon/Monitor.cc

```diff
@@ -1849,13 +1864,23 @@ void Monitor::handle_probe_probe(MonOpRequestRef op)
 
   dout(10) << "handle_probe_probe " << m->get_source_inst() << *m
 	   << " features " << m->get_connection()->get_features() << dendl;
-  uint64_t missing = required_features & ~m->get_connection()->get_features();
+  uint64_t missing =
+    get_quorum_con_features() & ~m->get_connection()->get_features();
```

Comment:
I don't understand why ``quorum_con_features`` is here.  We are probing, so there is no quorum--these features are only set in win_election and lose_election.